system/jlib/jfile.hpp

```diff
@@ -491,6 +491,19 @@ inline char getPathSepChar(const char *dir)
     return s?(*s):((*dir&&(dir[1]==':'))?'\\':PATHSEPCHAR);
 }
 
+inline char getPathSepCharEx(const char *s)
+{
+    bool foundBackslash=false;
+    if (s) while (*s) {
```

Comment:
I see that the code is cloned and modified, but...
non standard nesting of if/while and bracket placement.  Probably better with the definition in the c++ file (reduces compile times and complication of the header.